frontend/src/pages/intake-form/validation.ts

```diff
@@ -170,4 +170,26 @@ export const intakeFormValidationSchema = Yup.object().shape({
   supervisorLastName: Yup.string().required("Supervisor's last name is required"),
   supervisorEmail: Yup.string().required("Supervisor's email is required"),
   supervisorPhone: Yup.string(),
+
+  liaisonUnknown: Yup.string(),
+  liaisonFirstName: Yup.string(),
```

Comment:
Custom validation message on these fields is needed. You can also set .optional()

Reply:
Anything you think is fine, we just want to show a nice message incase they try to input non string values in the optional fields.

Reply:
Hrm okay as long as there is not weird looking validation msg displayed then we should be g2g